xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-test/xwiki-platform-livedata-test-pageobjects/src/main/java/org/xwiki/livedata/test/po/TableLayoutElement.java

```diff
@@ -450,7 +450,8 @@ public void filterColumn(String columnLabel, String content, boolean wait)
         if (classes.contains("filter-list")) {
             if (element.getAttribute(CLASS_HTML_ATTRIBUTE).contains("selectized")) {
                 SuggestInputElement suggestInputElement = new SuggestInputElement(element);
-                suggestInputElement.sendKeys(content).selectTypedText();
+                suggestInputElement.clearSelectedSuggestions();
+                suggestInputElement.sendKeys(content).selectByVisibleText(content);
```

Comment:
I think you're missing a wait for the suggestions to be fetched from the server and displayed.